### DISCUSSION TITLE
Clipboard not working

### DISCUSSION THREAD

**Discussion by Mukul-raii** [end-4/dots-hyprland#769] [Q&A]
i installed dots-hyprland in cachy os but the copy paste is not working properly i am unnable to paste  text from edge to vscode .  but if   i copy from vscode and passste in vscode its workding fine

**Comment by JxJxxJxJ** [comment]
Sorry if months late but it seems it's a common issue to copy things between xwayland and wayland apps, I'm not sure what could be the fix though
